example/sofast_fringe/example_process_single_facet.py

```diff
@@ -17,25 +17,33 @@
 import opencsp.common.lib.tool.log_tools as lt
 
 
-def example(dir_save: str):
-    """Example SOFAST script
+def example_process_single_facet():
+    """Example Sofast script
 
     Performs processing of previously collected Sofast data of single facet mirror.
-        1. Loads saved "single-facet" SOFAST collection data
-        2. Processes data with SOFAST
-        3. Prints best-fit parabolic focal lengths
-        4. Plots slope magnitude, physical setup
+        1. Loads saved single facet Sofast collection data

```

Comment:
For particularly long or complicated functions, I also like to describe the steps just like this.

My own personal style is to take this a step further; I add comments in the code with the step numbers. For example:

```python
# 1. Load data
...
# 2. Process
...
# 3. Calculate and log focal length from parabolic fit
...
# 4. Plot slope map
```

Reply:
Yes, I like it! That would make the examples much easier to follow! Inline numbers are now added.